.github/workflows/generateChangelog.yml

```diff
@@ -0,0 +1,32 @@
+name: Generate Changelog
+
+on:
+  workflow_call:
+
+jobs:
+  generate_changelog:
+    name: 'Generate Changelog'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: 'develop'
+          ssh-strict: false
+          token: ${{ secrets.IDEE_GH_TOKEN }}
+          fetch-depth: '0'
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+      - uses: ./.github/actions/gitConfig
+        with:
+          email: ${{ secrets.IDEE_GH_EMAIL }}
+      - name: Set NPM at the correct version for Lerna
+        run: npm install -g npm@9.8.1
+      - run: npm ci
+      - run: npm install -g shelljs && npm install -g lerna@5.5.4
```

Comment:
Do we use lerna in the changelog script?

Reply:
`salesforcedx-vscode/scripts/create-release-branch.js`